### ISSUE TITLE
Map links for some locations don't work

### ISSUE THREAD

**Issue by equivalentideas** [JasonThomasData/DocLink#22]
Try the following the “View map” links on this page http://hidden-hamlet-42593.herokuapp.com/en/doctors?q=Liverpool%2C+NSW

They don't take you to the proper address. But then on the google maps page you hit search, you get taken closer. Why is this? Can we fix it?

I think that having the name of the Doctor’s clinic in the address field might be causing this problem. Or maybe the way we link to maps is the problem—maybe there's a better url to send people to?

@JasonThomasData could you try and get to the bottom of this one?

**Comment by JasonThomasData** [comment]
I agree, it's definately the surgery name.
I think we should drop the doctor surgery name.
`I think the 'find my location' button would be useful here.` - actually scrap that, brain fart, unrelated issue
We should alter the doctors helper to give Google the address only.

**Comment by JasonThomasData** [comment]
I was just looking at the schema, this might be a simple matter of putting surgery name in a different field, rather than having the doctor surgery name in the address.
That way, we could leave the doctors view and doctors helper alone.

To reflect this in the design, perhaps we can put the surgery name (if we have one) in a subtext below the doctor's name?
I think that could be handled in the doctors view actually. 

What do you think about that?

**Comment by JasonThomasData** [comment]
This was partially an issue with [this bug](https://github.com/JasonThomasData/DocLink/issues/25), but we should still try out the schema issue. I'll have a go now.

**Comment by JasonThomasData** [comment]
This should be fixed now. I've added a change to the view so the surgery name appears separate from the address. 
See [this commit](https://github.com/JasonThomasData/DocLink/commit/102219d98ac741db6a66fb4707aab3ec76571036)

I also added a migration to split the address field. Now, the surgery name (if there is one added), is separate from the address.
See [this commit](https://github.com/JasonThomasData/DocLink/commit/7425c2af6e529972b0d718bc90c88e6e72e0dddb)
Now, Google only needs to deal with the address and we're not passing in the surgery name.
